app/views/devise/registrations/show.html.erb

```diff
@@ -25,6 +25,10 @@
     <dt style="width:165px"><strong><%= model_class.human_attribute_name(:favourite_sports) %>:</strong></dt>
     <dd><%= @user.favourite_sports %></dd>
   <% end %>
+  <% if @user.contact_information.present? %>
+      <dt style="width:165px"><strong><%= model_class.human_attribute_name(:contact_information) %>:</strong></dt>
```

Comment:
the indentation differs from the rest...

Reply:
Yes :)
But what are you doing here?

Reply:
Wobei dich womöglich der Inhalt der yaml Dateien interessiert :D